server/chat/backend/agent/skills/registry.py

```
@@ -0,0 +1,557 @@
+                {
+                    "service": r[0],
+                    "environment": r[1],
+                    "result": r[2],
+                    "build_number": r[3],
+                    "commit_sha": r[4],
+                    "trace_id": r[5],
+                    "received_at": str(r[6]) if r[6] else "?",
+                }
+                for r in rows
+            ]
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)